.github/workflows/basic-continuous-integration.yml

```diff
@@ -31,9 +35,7 @@ jobs:
         run: vendor/bin/phpunit --coverage-clover=coverage.xml
 
       - name: Monitor coverage
-
         if: github.event_name == 'pull_request'
-
         uses: slavcodev/coverage-monitor-action@1.1.0
         with:
```

Comment:
Can we add something like `comment_context: PHP ${{ matrix.php }} Coverage Report` so that the coverage report indicates which PHP version it was for in the GitHub comment?